packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,682 @@
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "PIN code",
+  "default": false,
+  "settings": {}
+}' "https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles"
+```
+
+#### Response Types
+
+HTTP 204:
+[Factor Profile Object](#factor-profile-object)
+
+## Factor Profile Feature Operations
+
+### Get Factor Profile Features
+
+<ApiOperation method="get" url="/api/v1/org/factors/${factorName}/profiles/${profileId}/features" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}/features"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Feature Object](#factor-profile-feature-object)
+
+### Get a Factor Profile Feature
+
+<ApiOperation method="get" url="/api/v1/org/factors/${factorName}/profiles/${profileId}/features/${featureId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+| `featureId`  | String | The factor profile feature's unique identifier. |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}/features/{featureId}"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Feature Object](#factor-profile-feature-object)
+
+### Update a Factor Profile Feature
+
+<ApiOperation method="put" url="/api/v1/org/factors/${factorName}/profiles/${profileId}/features/${featureId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The name of the factor type the profile is for. |
+| `profileId`  | String | The factor profile's unique identifier.         |
+| `featureId`  | String | The factor profile feature's unique identifier. |
+
+#### Request Example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "adoption",
+  "cardinality": {
+    "min": 0,
+    "max": 1
+  }
```

Comment:
I think there is now a selfService attribute that is required when you PUT an adoptionFeature.
Also, are we saying this API is EA? I am not sure we have concluded on all the possible objects that could be required for an adoption feature or any feature for that matter.
@okta/idx to confirm

Reply:
cc @iancorcoran-okta

Reply:
@AjayThomas-okta we are currently only doc'ing these endpoints for OIE LEA. So afaik it'll be shared in a limited manner with LEA customers and they'd know these will be in Beta state (therefore potential backward incompatible changes to them)

Reply:
Self Service attribute is required here, e.g.:

```
"selfService": {
    "eligibility": "ALLOWED",
    "verificationMethod": {
        "type": "ANY_FACTOR"    
     }
}
```

Eligibility can be ALLOWED or NOT_ALLOWED. Verification Method uses same model as existing /verificationMethods API but should only allow a chain that's a single level deep.